src/Middleware/AuthenticationMiddleware.php

```diff
@@ -158,6 +158,11 @@ protected function getRedirectUrl($target, ServerRequestInterface $request)
             $query = '?' . $query;
         }
 
+        if (($hashPos = strpos($target, '#')) !== false) {
```

Comment:
No inline assignment please. A separate line is fine.

Reply:
Corrected